### PR TITLE
Integrate with api

### PR DESCRIPTION
**Pull request by Gamaranto**
**Motivation**
This PR integrates the new API calls with umaverse UI. It does not however, merge the testing page into the address page 

**Details**
The API client has been refactored to accomodate for the new API, the `[address]` page now can distinguish between LSPs and EMPs and will render different views based on that.
**Testing**

Check a box to describe how you tested these changes and list the steps for reviewers to test.

- [ ]  Ran end-to-end test, running the code as in production
- [ ]  New unit tests created
- [ ]  Existing tests adequate, no new tests required
- [x]  All existing tests pass
- [ ]  Untested

### CODE REVIEW

File: utils/umaApi.ts [hunk 1]
```diff
@@ -70,39 +71,90 @@ export type EmpState = {
...
 
 type GetPriceSlice = (address: string) => Promise<string[]>;
 
-const getEmpsAddresses: GetEmpsAddresses = () => request("listEmpAddresses");
-const getEmpState: GetEmpState = (address) => request("getEmpState", address);
-const getEmpsState: GetEmpsState = () =>
-  getEmpsAddresses().then((addresses) =>
-    Promise.all(addresses.map(getEmpState))
-  );
+const getAddresses: GetAddresses = async () => {
```
Comment (daywiss) [inline utils/umaApi.ts:146]: didnt know you needed this, its another call we can add

File: utils/umaApi.ts [hunk 2]
```diff
@@ -70,39 +71,90 @@ export type EmpState = {
...
+  return [...empAddresses, ...lspAddresses];
+};
+
+const getState: GetState = async (address) => {
+  const state: any = await request("global/getState", address);
+  return state;
+};
 
-const getEmpStats: GetEmpStats = async (address) => {
+const getSynthStats: GetSynthStats = async (address) => {
```
Comment (daywiss) [inline utils/umaApi.ts:157]: i think theres going to be a problem with these calls. The specific calls are not exposed for all types on the global channel.  You need to know the type of the contract to get tvl/tvm from. EMPS have tvm and tvl, whle LSPS only have tvm.  The global channel only has combined tvl of all contracts, not per address, though its possible we can add this

**Review by daywiss — APPROVED** [review]
looks good, will need to fix some api things but we can do this in future pr. going to approve so @Tulun can start integrating

**Review by Tulun — COMMENTED** [review]
Just some nits from what I can tell.

**Pull request merged**